tests/integration/app/test_load_manager.py

```diff
@@ -0,0 +1,192 @@
+from copy import copy
+from datetime import datetime, timedelta
+from unittest.mock import patch
```

Comment:
I'm not sure that this is an integration test, I feel that it is more of a unit test since all outside integration is mocked. If it was hitting something that resembled/copied cloudfoundry functionality then this would be an integration test...

Reply:
Good suggestion. I'm going to rewrite the tests, particularly the first two, so we're inheriting the interface rather than mocking it, and we are just mocking the CF library dependency's responses.